R/stochastic.R

```diff
@@ -0,0 +1,78 @@
+#' Get data from stochastic tables with specified grouping and filters
+#'
+#' Can query from cross_all, cross_under5, cohort_all and cohort_under5
+#'
+#' @param annex Connection to annex db
+#' @param table One of cross_all, cross_under5, cohort_all or cohort_under5
+#' @param groups Categories to group by for aggregating in query
+#' @param filters Filters to apply before aggregation
+#'
+#' @return Mean, 0.025 and 0.975 quantiles for deaths_default, deaths_novac.
+#' deaths_impact, dalys_default, dalys_novac, dalys_impact with specified
+#' groupings.
+#' @export
+get_stochastic_data <- function(annex, table,
```

Comment:
I wonder if `fetch_stochastic_data` might be just a touch more descriptive?

Reply:
Yep sounds good